projects/npm-tools/packages/npm-scripts/package.json

```diff
@@ -76,7 +76,7 @@
 		"stylelint": "^13.2.0",
 		"terser": "5.3.8",
 		"ts-loader": "^8.0.4",
-		"typescript": "^4.2.3",
+		"typescript": "^4.4.0",
```

Comment:
Should we prefer a fixed version `4.4.2` here or at least `~4.4.2`? After all, `^4.2.3` also works for `4.4.2` ;) 

Based on the experience here, I get the feeling we want to be very explicit about which specific version we're using (and shipping). That way, devs in Liferay DXP might have an easier time figuring out which TS options are available to them.

This makes me think that we might also want to start being more vocal about TS support so we get more people on board, but maybe that can wait until we land some bits of https://github.com/liferay-frontend/liferay-portal/pull/1379

Reply:
Totally agree. Just pushed an update